packages/snap/src/index.ts

```diff
@@ -106,6 +110,8 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ origin, request }) =>
             overrideFeeCurrency,
             network.name,
           );
+        } else {
+          return;
```

Comment:
so if  no  overrideFeeCurrency  is given it will return here and not send the transaction

Reply:
Yes updated now.

Reply:
basically 
Disapprove = null
Approve w/o inputing any value = undefined
Approve with inputing a value = value

![image](https://github.com/celo-org/gas-snap/assets/8665738/0a3600bc-d384-471c-b110-924a772beee5)